examples/ex24.cpp

```diff
@@ -45,8 +47,11 @@ using namespace mfem;
 double p_exact(const Vector &x);
 void gradp_exact(const Vector &, Vector &);
 double div_gradp_exact(const Vector &x);
+void v_exact(const Vector &x, Vector &v);
+void curlv_exact(const Vector &x, Vector &cv);
 
 int dim;
+double freq = 1.0, kappa;
```

Comment:
How high can you go before a direct solver is needed?

Reply:
This example is very simple in terms of solvers. The only solver is for a mass matrix, which can use CG with Jacobi preconditioning. It should converge fast even for high frequency.